src/main/java/org/spongepowered/common/mixin/core/advancement/MixinAdvancementList.java

```diff
@@ -70,18 +72,13 @@ private void onLoadAdvancements(Map<ResourceLocation, Advancement.Builder> advan
         if (!SpongeImplHooks.isMainThread() || !ShouldFire.GAME_REGISTRY_EVENT_REGISTER) {
             return;
         }
-
+        final SpongeGameRegistryRegisterEvent<org.spongepowered.api.advancement.Advancement> event =
+                new SpongeGameRegistryRegisterEvent<>(Cause.of(EventContext.empty(), SpongeImpl.getRegistry()),
```

Comment:
Why are you creating the cause yourself and not delegating to the cause stack manager?

Reply:
Which, in this case, you'd still want to put it in the try statement so the ListenerPhaseContext has a chance to provide some additional context

Reply:
It's a mistake, so I will fix it